### ISSUE TITLE
Python fails to build due to pycurl 404 and wrong digest file

### ISSUE THREAD

**Issue by GldRush98** [SynoCommunity/spksrc#2569]
### Setup
Package Name: spk/Python
Package Version: 2.7.13-16

NAS Model: DS1512+
NAS Architecture: x86
DSM version: DSM 6.0.2-8451 Update 6

### Expected behavior
Python should build cleanly with out erros

### Actual behavior
Python fails to build to to pycurl's download url being 404'ed.

### Steps to reproduce
1. cd spk/python
2. make arch-x86

#### Package log
```
running install_egg_info
running egg_info
creating pycrypto.egg-info
writing pycrypto.egg-info/PKG-INFO
writing top-level names to pycrypto.egg-info/top_level.txt
writing dependency_links to pycrypto.egg-info/dependency_links.txt
writing manifest file 'pycrypto.egg-info/SOURCES.txt'
warning: manifest_maker: standard file '-c' not found

reading manifest file 'pycrypto.egg-info/SOURCES.txt'
reading manifest template 'MANIFEST.in'
writing manifest file 'pycrypto.egg-info/SOURCES.txt'
Copying pycrypto.egg-info to /spksrc/spk/python/work-x86-5.2/wheelbuild/pycrypto-2.6.1-py2.7.egg-info
running install_scripts
creating /spksrc/spk/python/work-x86-5.2/wheelbuild/pycrypto-2.6.1.dist-info/WHEEL
===>  Installing for pycrypto
find /spksrc/spk/python/work-x86-5.2/install//usr/local/python/ \! -type d -printf '%P\n' | sort > /spksrc/spk/python/work-x86-5.2/pycrypto.plist.tmp
find /spksrc/spk/python/work-x86-5.2/install//usr/local/python/ \! -type d -printf '%P\n' | sort | \
  diff /spksrc/spk/python/work-x86-5.2/pycrypto.plist.tmp -  | grep '>' | cut -d' ' -f2- > /spksrc/spk/python/work-x86-5.2/pycrypto.plist
make[2]: Leaving directory '/spksrc/cross/pycrypto'
make[2]: Entering directory '/spksrc/cross/pycurl'
===>  Downloading files for pycurl
wget http://pycurl.sourceforge.net/download/pycurl-7.43.0.tar.gz
http://pycurl.sourceforge.net/download/pycurl-7.43.0.tar.gz:
2016-12-23 16:12:38 ERROR 404: Not Found.
../../mk/spksrc.download.mk:56: recipe for target 'download_target' failed
make[2]: *** [download_target] Error 8
make[2]: Leaving directory '/spksrc/cross/pycurl'
../../mk/spksrc.depend.mk:44: recipe for target 'depend_target' failed
make[1]: *** [depend_target] Error 2
make[1]: Leaving directory '/spksrc/spk/python'
../../mk/spksrc.spk.mk:416: recipe for target 'arch-x86' failed
make: [arch-x86] Error 2 (ignored)
root@synocommunity-spksrc1:/spksrc/spk/python#
```

This is a relatively minor issue.  I thought I had more python build issues, but I wiped the docker image I was using and clean installed everything, and I finally got python to build all the way through.
The one issue I did have was with pycurl giving a 404 error, and the digest file for it using the old version hashes.  I was able to plunk in the new url which seems totally different from the old one, as well as update the digest file and was able to build successfully.
Just thought I'd let you know I got it to build, but did have this one minor issue.
Thank you :)

**Comment by Dr-Bean** [comment]
You're right :) Seems the URL changed to https://dl.bintray.com/pycurl/pycurl/pycurl-7.43.0.tar.gz, and I forgot to update the digests.
Having said that, it's probably even better to download directly from PyPI: `PKG_DIST_SITE = https://files.pythonhosted.org/packages/source/p/pycurl` should do fine (along with a digests update of course)
Did you plan/want to create a PR with the changes?

**Comment by GldRush98** [comment]
Ok, I created a PR.  I hope I did it right :+1:

**Comment by Dr-Bean** [comment]
Merged with 6e0caa47f370357dc478c6d41e195c68948965ff. Thanks :)
